packages/core/src/app/payment/paymentMethod/PaymentMethodTitle.tsx

```diff
@@ -5,20 +5,19 @@ import React, { FunctionComponent, memo, ReactNode } from 'react';
 
 import { withLanguage, WithLanguageProps } from '@bigcommerce/checkout/locale';
 import { CheckoutContextProps , PaymentFormValues } from '@bigcommerce/checkout/payment-integration-api';
+import { BraintreePaypalCreditBanner, PaypalCommerceCreditBanner } from '@bigcommerce/checkout/paypal-utils';
 
 import { withCheckout } from '../../checkout';
 import { connectFormik, ConnectFormikProps } from '../../common/form';
 import { isExperimentEnabled } from '../../common/utility';
 import { CreditCardIconList, mapFromPaymentMethodCardType } from '../creditCard';
 
-import BraintreePaypalCreditDescription from './BraintreePaypalCreditDescription';
```

Comment:
Is it intentional that we still keep the files in `core`?

Reply:
Removed. @bc-peng take a look pls